OneginiExampleAppSwift/ViewControllers/Profile/ProfileViewController.xib

```diff
@@ -239,7 +207,6 @@
                 <constraint firstItem="L2S-8H-peg" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="20" id="ryR-0y-1XE"/>
                 <constraint firstItem="L8M-xf-xKf" firstAttribute="top" secondItem="Hr6-Gb-jxi" secondAttribute="bottom" constant="22" id="tAk-Qm-vZg"/>
             </constraints>
-            <viewLayoutGuide key="safeArea" id="fnl-2z-Ty3"/>
```

Comment:
why this?

Reply:
Hmm perhaps I mess with the xib too much. Let me redo carefully.

Reply:
Wait, it is still there, have a look at line 196.